voluncheer/voluncheer/urls.py

```diff
@@ -18,32 +18,16 @@
 from django.urls import include
 from django.urls import path
 
-from profiles.views.home import SignUpView
-from profiles.views.organizations import OrganizationSignUpView
-from profiles.views.volunteers import VolunteerSignUpView
 from voluncheer import settings
 from voluncheer.environment import environment
 
 urlpatterns = [
-    path("__debug__/", include("debug_toolbar.urls")),
-    path("admin/", admin.site.urls),
     path("", include("profiles.urls")),
     path("accounts/", include("django.contrib.auth.urls")),
-    path("accounts/signup/", SignUpView.as_view(), name="signup"),
-    path(
-        "accounts/signup/organization/",
-        OrganizationSignUpView.as_view(),
-        name="organization_signup",
-    ),
-    path(
-        "accounts/signup/volunteer/",
-        VolunteerSignUpView.as_view(),
-        name="volunteer_signup",
-    ),
-    # Unimplemented urls.
-    # path("chat/", include("chatroom.urls")),
-    path("opportunityboard/", include("opportunityboard.urls")),
+    path("admin/", admin.site.urls),
+    path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
I think you'll want to only include this when `settings.DEBUG` is true. If not, what happens when `settings.DEBUG` is false and someone visits this url?

Reply:
So, if `settings.DEBUG` is false, `debug_toolbar ` is not added to installed apps. So if someone enters that url, it doesn't find anything because `debug_toolbar.urls `doesn't exist. So we get a standard 404 error.

Interestingly, we get a 404 for this url as well if debug mode is on, so this url must just be something the toolbar needs to render as opposed to an actual page.

Reply:
Hmm, that's interesting. Thanks for figuring it out. I assume we would still only want to include it when `settings.DEBUG` is true though. We don't know what we are risking by including the URL in a non-debug environment.

Reply:
Sounds good, I made the requested change in urls.py